generator/generator.go

```diff
@@ -204,11 +204,16 @@ func NewGenerator(options Options) (*Generator, error) {
 		interfaceType = options.InterfaceName
 		srcPackageAST.Name = ""
 	} else {
+		_, imps, _ := iterateFiles(srcPackageAST, options.InterfaceName)
+		srcPackageAlias := getSrcPackageAlias(imps, srcPackage.PkgPath)
+
 		if options.SourcePackageAlias != "" {
 			srcPackageAST.Name = options.SourcePackageAlias
+		} else if srcPackageAlias != "" {
+			srcPackageAST.Name = srcPackageAlias
 		}
 
-		options.Imports = append(options.Imports, `"`+srcPackage.PkgPath+`"`)
+		options.Imports = append(options.Imports, srcPackageAlias+` "`+srcPackage.PkgPath+`"`)
```

Comment:
here the best way is to always use srcPackageAST.Name for package alias. But im afraid we would break backwards compatibility because e.g. minimock has already such import that was added manually https://github.com/gojuno/minimock/blob/master/template.go#L22 . So if we add here import with srcPackageAST.Name alias we would break existing templates because we would have the same import duplicated